tests/puppeteer/campaigns/functional/BO/14_advancedParameters/06_team/profiles/06_profilesPagination.js

```diff
@@ -0,0 +1,147 @@
+require('module-alias/register');
+const {expect} = require('chai');
+const helper = require('@utils/helpers');
+const loginCommon = require('@commonTests/loginBO');
+const ProfileFaker = require('@data/faker/profile');
+// Importing pages
+const BOBasePage = require('@pages/BO/BObasePage');
+const LoginPage = require('@pages/BO/login');
+const DashboardPage = require('@pages/BO/dashboard');
+const EmployeesPage = require('@pages/BO/advancedParameters/team/index');
+const ProfilesPage = require('@pages/BO/advancedParameters/team/profiles/index');
+const AddProfilePage = require('@pages/BO/advancedParameters/team/profiles/add');
+// Test context imports
+const testContext = require('@utils/testContext');
+
+const baseContext = 'functional_BO_advancedParameters_team_profiles_paginationProfiles';
+
+let browser;
+let page;
+let numberOfProfiles = 0;
+const profileData = new ProfileFaker();
+
+// Init objects needed
+const init = async function () {
+  return {
+    boBasePage: new BOBasePage(page),
+    loginPage: new LoginPage(page),
+    dashboardPage: new DashboardPage(page),
+    employeesPage: new EmployeesPage(page),
+    profilesPage: new ProfilesPage(page),
+    addProfilePage: new AddProfilePage(page),
+  };
+};
+
+describe('Profiles pagination', async () => {
+  // before and after functions
+  before(async function () {
+    browser = await helper.createBrowser();
+    page = await helper.newTab(browser);
+    this.pageObjects = await init();
+  });
+  after(async () => {
+    await helper.closeBrowser(browser);
+  });
+  // Login into BO
+  loginCommon.loginBO();
+
+  it('should go to \'Advanced parameters>Team\' page', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'goToAdvancedParamsPage', baseContext);
+    await this.pageObjects.boBasePage.goToSubMenu(
+      this.pageObjects.boBasePage.advancedParametersLink,
+      this.pageObjects.boBasePage.teamLink,
+    );
+    await this.pageObjects.boBasePage.closeSfToolBar();
+    const pageTitle = await this.pageObjects.employeesPage.getPageTitle();
+    await expect(pageTitle).to.contains(this.pageObjects.employeesPage.pageTitle);
+  });
+
+  it('should go to \'Profiles\' page', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'goToProfilesPage', baseContext);
+    await this.pageObjects.employeesPage.goToProfilesPage();
+    const pageTitle = await this.pageObjects.profilesPage.getPageTitle();
+    await expect(pageTitle).to.contains(this.pageObjects.profilesPage.pageTitle);
+  });
+
+  it('should reset all filters and get number of profiles', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'resetFilterFirst', baseContext);
+    numberOfProfiles = await this.pageObjects.profilesPage.resetAndGetNumberOfLines();
+    await expect(numberOfProfiles).to.be.above(0);
+  });
+  // 1 : Create 11 pages
+  /* eslint-disable no-loop-func */
```

Comment:
Same comment as another PR, use one of the workaround there https://eslint.org/docs/rules/no-loop-func or either disable it in your eslint configuration if this is boring for you!

Reply:
fixed, @nesrineabdmouleh see https://github.com/PrestaShop/PrestaShop/pull/19074/commits/a7085c672a7c2e571b98b6d99c2acb941e8817f3

Reply:
Done thx :)